plugins/httpapi/nd.py

```diff
@@ -236,14 +236,15 @@ def send_file_request(self, method, path, file=None, data=None):
             self.error = dict(code=self.status, message='ND HTTPAPI create data field Exception: {0} - {1}'.format(e, traceback.format_exc()))
             raise ConnectionError(json.dumps(self._verify_response(None, method, path, None)))
 
-        try: 
+        try:
             # create fields for MultipartEncoder
             fields = dict(data=('data.json', data_str, 'application/json'), file=(os.path.basename(file), open(file, 'rb'), mimetypes.guess_type(file)))
             mp_encoder = MultipartEncoder(fields=fields)
-            self.headers['Content-Type'] = mp_encoder.content_type
-            self.headers['Accept'] = '*/*'
-            self.headers['Accept-Encoding'] = 'gzip, deflate, br'
-            response, rdata = self.connection.send(path, mp_encoder.to_string(), method=method, headers=self.headers)
+            multiheader = {}
```

Comment:
why not write as dict literal? would be same as self.header declaration and would be 1 line